src/api/pack/xdsl/pack-xdsl.v6.service.js

```diff
@@ -77,5 +77,15 @@ angular.module('ovh-api-services').service('OvhApiPackXdslV6', ($resource, OvhAp
         packName: '@packName',
       },
     },
+    getContactOwner: {
+      method: 'GET',
+      url: '/pack/xdsl/:packName/contactOwner',
```

Comment:
API returns a type `pack.xdsl.ContactInfos` with the following properties:

```js
{
    address
    city
    country
    email
    firstname
    name
    organisation
    phone
    phoneCountry
    zip
}
```

Are you sure about the `transformResponse`?

Reply:
Yes I'm sure.